torchvision/utils.py

```diff
@@ -88,6 +89,7 @@ def norm_range(t, value_range):
         else:
             norm_range(tensor, value_range)
 
+    assert isinstance(tensor, torch.Tensor)
```

Comment:
Type check complaints about
> Item "List[Tensor]" of "Union[Tensor, List[Tensor]]" has no attribute "size"

Since list of tensors should have been stacked together on line 57, I think this is a fair assertion.

Reply:
Thanks @kazhang, clearly this error is not related to your changes.

@pmeier I tagged you because of the git-blame info. The `tensor: Union[torch.Tensor, List[torch.Tensor]]` is indeed defined as list. My understanding is that this won't work, can you confirm?

Reply:
Nope, this works as expected: 

https://github.com/pytorch/vision/blob/f5dca445a2a22945a3a34fbf9abe409e23f83fc5/torchvision/utils.py#L54-L56

I look into it an see why `mypy` doesn't pick up on this.

Reply:
Sorry I looked it quickly and missed this. Sounds good.

Reply:
Keeping this open for now. I'm fine with the fix for this PR, but something weird is going on. I can reproduce the issue, but this PR shouldn't do anything to trigger it. I'll investigate more thoroughly.

Reply:
@pmeier just to clarify, should I remove this assertion and ignore the type check error for now? since you'll investigate more thoroughly

Reply:
Keep it in for now. I'll try to figure out how to make `mypy` happy without the `assert` in.